cmd/janus/init.go

```diff
@@ -58,6 +63,26 @@ func initDistributedTracing() io.Closer {
 	return closer
 }
 
+func initCircuitBreaker() {
+	// Setup metrics for circuit breaker
+	c, err := plugins.InitializeStatsdCollector(&plugins.StatsdCollectorConfig{
+		StatsdAddr: globalConfig.Stats.DSN,
+		Prefix:     globalConfig.Stats.Prefix,
+	})
+
+	metricCollector.Registry.Register(c.NewStatsdCollector)
+	if err != nil {
+		log.WithError(err).Error("Error initializing statsd client for circuit breaker")
+	}
+
+	// Setup hystrix dashboard stream
+	if globalConfig.CircuitBreaker.DashboardEnabled {
+		hystrixStreamHandler := hystrix.NewStreamHandler()
+		hystrixStreamHandler.Start()
+		go http.ListenAndServe(net.JoinHostPort("", "81"), hystrixStreamHandler)
```

Comment:
port should be configurable

Reply:
Yes, it should be configurable I. The circuit breaker config section